### PR TITLE
Wip/hughsie/no auto remove

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by krankun** [comment]
we got the  retimer nvm updates working with this change :)

localhost ~ # fwupdmgr install /home/root/retimer-2.21.cab --allow-reinstall --force --verbose
(fwupdmgr:4317): GLib-DEBUG: 09:03:05.140: setenv()/putenv() are not thread-safe and should not be used after threads are created
(fwupdmgr:4317): GLib-GIO-DEBUG: 09:03:05.141: _g_io_module_get_default: Found default implementation dummy (GDummyProxyResolver) for ?gio-proxy-resolver?
(fwupdmgr:4317): Fwupd-DEBUG: 09:03:05.145: Emitting ::status-changed() [idle]
(fwupdmgr:4317): Fwupd-DEBUG: 09:03:05.150: Emitting ::status-changed() [decompressing]
Decompressing?           [ -                                     ](fwupdmgr:4317): Fwupd-DEBUG: 09:03:05.160: Emitting ::status-changed() [idle]
Decompressing?           [***************************************]
(fwupdmgr:4317): Fwupd-DEBUG: 09:03:05.160: Emitting ::status-changed() [waiting-for-auth]
Authenticating?          [ -                                     ]
(fwupdmgr:4317): Fwupd-DEBUG: 09:03:08.823: Emitting ::status-changed() [device-restart]
Restarting device?       [  -                                    ](fwupdmgr:4317): Fwupd-DEBUG: 09:03:08.823: Emitting ::status-changed() [device-write]
Writing?                 [  -                                    ](fwupdmgr:4317): Fwupd-DEBUG: 09:03:08.824: Emitting ::status-changed() [decompressing]
Decompressing?           [    \                                  ](fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.707: Emitting ::status-changed() [device-write]
Writing?                 [***************************************]
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.711: Emitting ::device-changed(16d35c510618116e7899ae45e62605572872e713)
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.711: Emitting ::device-changed(16d35c510618116e7899ae45e62605572872e713)
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.711: Emitting ::device-changed(16d35c510618116e7899ae45e62605572872e713)
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.735: Emitting ::device-changed(16d35c510618116e7899ae45e62605572872e713)
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.751: Emitting ::device-changed(16d35c510618116e7899ae45e62605572872e713)
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.768: Emitting ::device-changed(16d35c510618116e7899ae45e62605572872e713)
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.768: Emitting ::status-changed() [device-restart]
Restarting device?       [***************************************]
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.768: Emitting ::status-changed() [device-busy]
Waiting?                 [***************************************]
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.769: Emitting ::status-changed() [unknown]
(fwupdmgr:4317): Fwupd-DEBUG: 09:04:12.769: Emitting ::changed()
Successfully installed firmware

File: plugins/thunderbolt/fu-thunderbolt-controller.c [hunk 1]
```diff
@@ -151,7 +151,7 @@ fu_thunderbolt_controller_can_update(FuThunderboltController *self)
 }
 
 static gboolean
-fu_thunderbolt_controller_set_port_offline_cb(gpointer user_data)
+fu_thunderbolt_controller_set_port_online_cb(gpointer user_data)
```
Comment (superm1) [inline plugins/thunderbolt/fu-thunderbolt-controller.c:154]: I thought this was intentional, @krankun please confirm.

Comment (krankun) [reply plugins/thunderbolt/fu-thunderbolt-controller.c:154]: echo 1 > offline // its a offline operation
echo 0 > offline // its online operation 

so this change looks ok to me

**Comment by krankun** [comment]
once merged , we will back port to chromium , cc: @djcampello  , we have the nvm updates working now for usb4 retimer

**Pull request merged**